web/app/(marketing)/marketing.module.css

```
@@ -0,0 +1,559 @@
+
+.regionDot {
+  position: absolute;
+  width: 14px;
+  height: 14px;
+  border-radius: 50%;
+  background: #0f6bff;
+  box-shadow: 0 0 0 8px rgba(15, 107, 255, 0.18);
+  animation: dotPulse 2.6s ease-in-out infinite;
+}
+
+@keyframes dotPulse {
```

Stylelint (17.9.0)

[error] Expected keyframe name "dotPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)